src/inputs/TextFieldBase.tsx

```diff
@@ -255,6 +255,11 @@ export function TextFieldBase<X extends Only<TextFieldXss, X>>(props: TextFieldB
                 }}
               />
             )}
+            {errorInTooltip && errorMsg && (
```

Comment:
Thought: Do you think we should reserve space for the icon, whenever `errorInTooltip` is true, so that the UI doesn't jump as the field flips between in-error/no-error/in-error?

Reply:
Hmm... I think we'd need to explore that a little more. We have other places where icons/adornments show up and leave, and we haven't had any reports about weird behavior yet. But would be good to check. Thanks for flagging.